ui/admin/app/components/form/worker/index.hbs

```diff
@@ -52,7 +52,36 @@
       </field.errors>
     {{/if}}
   </form.textarea>
-
+  <form.fieldset>
+    <:title>{{t 'form.worker_tags.label'}}</:title>
+    <:description>{{t 'form.worker_tags.help'}}</:description>
+    <:body>
+      <Rose::Table @style='condensed' as |table|>
+        <table.body as |body|>
+          {{#each-in @model.config_tags as |key value|}}
+            <body.row as |row|>
+              <row.cell>
+                <form.input
+                  @type='text'
+                  @value={{key}}
+                  @disabled={{true}}
+                  @readOnly={{true}}
+                />
+              </row.cell>
+              <row.cell>
+                <form.input
+                  @type='text'
+                  @value={{value}}
```

Comment:
Can we display the tags with spaces between the commas rather than just outputting the array?